### PR TITLE
chore(wren-ui): Show AI exception errors to client when deploying manifest

### PR DESCRIPTION
**Pull request by andreashimin**
## Description
Show deploying error toast, and remind the user to check the detail in logs.

## UI Screenshot
image

### CODE REVIEW

**Review by fredalai — APPROVED** [review]
LGTM

File: wren-ui/src/apollo/server/adaptors/wrenAIAdaptor.ts [hunk 1]
```diff
@@ -282,12 +278,13 @@ export class WrenAIAdaptor implements IWrenAIAdaptor {
         `${this.wrenAIBaseEndpoint}/v1/semantics-preparations/${deployId}/status`,
       );
       if (res.data.error) {
-        logger.debug(`deploy error: ${res.data.error}`);
+        // passing AI response error string to catch block
+        throw res.data.error;
```
Comment (wwwy3y3) [inline wren-ui/src/apollo/server/adaptors/wrenAIAdaptor.ts:282]: If `res.data.error` is not an error object. Could we wrap it with `Error` ? Maybe we can use our `Errors.create` function ?

Comment (andreashimin) [reply wren-ui/src/apollo/server/adaptors/wrenAIAdaptor.ts:282]: Maybe wrap it with `Error`? 
The error will be retrieved in the response body as the structure in the image from the deploy API.
image

Using `Errors.create` function currently seems unnecessary here.

Comment (wwwy3y3) [reply wren-ui/src/apollo/server/adaptors/wrenAIAdaptor.ts:282]: sure

**Review by wwwy3y3 — APPROVED** [review]
LGTM

**Pull request merged**